src/cloud/reference/git-integration.md

```diff
@@ -28,6 +28,7 @@ In addition to Git's requirements for [valid branch names](https://www.kernel.or
 
```

Comment:
@barnato-v  If the ASCII thing has been a bother, then maybe a clarification here (in the stem sentence) could solve the problem. Maybe change it to:

In addition to the branch naming requirements outlined in the [`git-check-ref-format` Manual Page](https://www.kernel.org/pub/software/scm/git/docs/git-check-ref-format.html), {{site.data.var.ece}} adds two additional requirements: